driver/normalizer/normalizer.go

```diff
@@ -243,3 +283,42 @@ var Normalizers = []Mapping{
 		},
 	)),
 }
+
+type singleQuote struct {
+	op Op
+}
+
+func (op singleQuote) Kinds() nodes.Kind {
+	return nodes.KindString
+}
+
+func (op singleQuote) Check(st *State, n nodes.Node) (bool, error) {
+	sn, ok := n.(nodes.String)
+	if !ok {
+		return false, nil
+	}
+	s := string(sn)
+	if !strings.HasPrefix(s, `'`) || !strings.HasSuffix(s, `'`) {
+		return false, nil
+	}
+	s = s[1 : len(s)-1]
+	s, err := unquoteSingle(s)
+	if err != nil {
+		return false, err
+	}
+	return op.op.Check(st, nodes.String(s))
+}
+
+func (op singleQuote) Construct(st *State, n nodes.Node) (nodes.Node, error) {
+	n, err := op.op.Construct(st, n)
+	if err != nil {
+		return nil, err
+	}
+	sn, ok := n.(nodes.String)
+	if !ok {
+		return nil, ErrUnexpectedType.New(nodes.String(""), n)
+	}
+	s := string(sn)
```

Comment:
`return nodes.String(quoteSingle(string(sn))), nil`

Or at least `s := quoteSingle(string(sn))`, reassignment is really not needed here and makes my whiskers twitch. :)